### ISSUE TITLE
Show login screen after token expires

### ISSUE THREAD

**Issue by pastewka** [jic-dtool/dtool-lookup-webapp#5]
If the JWT token is set to expire, then the webapp no longer shows datasets after expiration. The user should probably be send back to the login screen once this happens.

**Comment by tjelvar-olsson** [comment]
Yes, good point. This defect entered the webapp in version 0.2.0. Prior to that all the data for the webapp was fetched in a single request when the user logged in. This method resulted in the webapp becoming unresponsive when it lookup server contained lots of datasets, and in particular where the datasets had lots of items in them. The code was therefore refactored to request the README and MANIFEST data when a user selected a dataset. However, no code was added for checking that token was still active or whether the request was successful.

**Comment by tjelvar-olsson** [comment]
Commit 214ed2b adds some more explanatory text and a button for logging out in

<img width="512" alt="logout_button" src="https://user-images.githubusercontent.com/10738286/81505779-b29eac80-92e9-11ea-8c7e-b72abc5ad262.png">
